tests/Unit/Domain/Creators/Python/Test_Sphere.py

```diff
@@ -9,6 +9,7 @@ class TestSphere(unittest.TestCase):
     def test_construction(self):
         sphere = Sphere(inner_radius=1.,
                         outer_radius=2.,
+                        inner_cube_sphericity=0.0,
```

Comment:
Can be `0.` to be consistent with lines 10 and 11

Reply:
Updated!